luigi/execution_summary.py

```diff
@@ -351,6 +353,8 @@ def _summary_format(set_tasks, worker):
     for status in _ORDERED_STATUSES:
         if status not in comments:
             continue
+        if status == 'ever_failed' and (set_tasks["ever_failed"] == set_tasks["failed"]):
```

Comment:
I think there's a bug here that can lead to crash. It seems like status can be `status=ever_failed`, and then you index in  `comments[status]`. Won't that crash? What about this case?

If you change this, please also make a test case testing this.